src/ensembl/production/metadata/updater/core.py

```diff
@@ -364,6 +390,46 @@ def new_genome(self, meta_session, species_id, organism, assembly, assembly_data
 
         return new_genome, assembly_genome_dataset, genebuild_genome_dataset
 
+    def _create_genome_group_members(self, meta_session, species_id, new_genome, planned_release):
+        """
+        Add genome to genome groups specified in meta keys.
+
+        Args:
+            meta_session: The metadata database session
+            species_id: The species ID from the core database
+            new_genome: The newly created Genome object
+            planned_release: The EnsemblRelease object
+
+        Raises:
+            MetadataUpdateException: If a specified genome group doesn't exist
+        """
+        genome_group_names = self.get_meta_all_values(species_id, "genome.genome_group")
+
+        if not genome_group_names:
+            return
```

Comment:
```suggestion
            return None
```
Being explicit is always better for readability.